static/js/components/EmploymentForm.js

```diff
@@ -16,6 +16,14 @@ import ProfileFormFields from '../util/ProfileFormFields';
 import ConfirmDeletion from './ConfirmDeletion';
 
 class EmploymentForm extends ProfileFormFields {
+  static propTypes = {
```

Comment:
oh, you know I'm not sure how that works. I can just move this onto `ProfileFormFields`